app/controllers/whois_records_controller.rb

```diff
@@ -13,9 +13,10 @@ def show
     respond_to do |format|
       format.json do
         if @whois_record
-          render :show, status: :ok
+          render :show, status: :ok if @whois_record
```

Comment:
What's the point in doubling the condition? If we are in this very branch of IF statement, we are already got the @whois_resord, I presume.

Reply:
You're right, no point at all. Thanks for pointing it out!